assets/together.yaml

```diff
@@ -109,3 +109,26 @@
   prohibited_uses: ''
   monitoring: ''
   feedback: ''
+
+- type: dataset
+  name: RedPajama-Data
+  organization: Together
+  description: The RedPajama base dataset is a 1.2 trillion token fully-open dataset
+    created by following the recipe described in the LLaMA paper
+  created_date: 2022-04-17
+  url: https://www.together.xyz/blog/redpajama
+  datasheet: https://huggingface.co/datasets/togethercomputer/RedPajama-Data-1T
+  modality: text
+  size: 1.2 trillion tokens
+  sample: []
+  analysis: ''
+  dependencies: [LLaMA dataset, GitHub, Wikipedia]
```

Comment:
The RedPajama dataset shouldn't depend on the LLaMA dataset. It was created to emulate or replicate the dataset (i.e. the developers attempted to collect a dataset that resembled the LLaMA dataset), but it doesn't literally depend on the dataset (i.e. LLaMA dataset is not a subset of the RedPajama dataset).